eos-storage.md

```diff
@@ -13,7 +13,62 @@ minutes: 10
 During a real analysis the output of your jobs will quickly grow
 beyond what fits onto your AFS space. CERN provides you with 2TB of
 space on a set of hard drives called the [EOS
-service](http://information-technology.web.cern.ch/services/eos-service).
+service](http://information-technology.web.cern.ch/services/eos-service) and
+a grid storage quota of 2TB.
+
+To retrieve a job outputfile, one can use three types of files :  
+- `LocalFile`: the standard one with the output file directly downloaded to
+the `gangadir`.  
+- `DiracFile`: the output file is stored directly on the grid and be accessed
+through XRootD protocol.  
+
+In this lesson, we will focus on the use of `DiracFile` in `ganga` to manage big
+output files.
+
+We can reuse what has been done to run [a DaVinci job on the
+grid](https://lhcb.github.io/first-analysis-steps/davinci-grid.html) and adapt
+the `j.outputfiles` part.
+
+To add the DiracFile in the configuration of the job we just need :
+```python:
+j = Job(application=DaVinci(version='v41r2'))
+j.backend = Dirac()
+j.name = 'First ganga job'
+j.inputdata = j.application.readInputData('data/MC_2012_27163003_Beam4000GeV2012MagDownNu2.5Pythia8_Sim08e_Digi13_Trig0x409f0045_Reco14a_Stripping20NoPrescalingFlagged_ALLSTREAMS.DST.py')
+j.application.optsfile = 'code/davinci-grid/ntuple_options_grid.py'
+j.outputfiles = [
+DiracFile('DVntuple.root')
+]
+```
+
+When the job is completed, no output is dowloaded but some interesting
+informations are provided by typing `j.outputfiles[0]` :
+
+```python:
+DiracFile(namePattern='DVntuple.root',
+lfn='/lhcb/user/a/another/2016_11/146255/146255492/DVntuple.root',
```

Comment:
I would indent these two lines to match `namePattern`, or put `namePattern` in the next line with 4 spaces:
```python
DiracFile(namePattern='DVntuple.root',
          lfn='...',
          localDir='...')